manifests/s/ScreamingFrog/SEOSpider/16.4/ScreamingFrog.SEOSpider.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=QUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: ScreamingFrog.SEOSpider
+PackageVersion: "16.4"
+PackageLocale: en-US
+Publisher: Screaming Frog Ltd
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: Screaming Frog SEO Spider
+# PackageUrl: 
+License: Copyright (c) Screaming Frog Ltd.
+# LicenseUrl: 
+# Copyright: 
+# CopyrightUrl: 
+ShortDescription: The Screaming Frog SEO Spider is a fast and advanced SEO site audit tool.
+# Description: 
+# Moniker: 
+# Tags: 
+# Agreements: 
+# ReleaseNotes: 
+# ReleaseNotesUrl: 
+ManifestType: defaultLocale
+ManifestVersion: 1.2.0
```

```suggestion
ManifestVersion: 1.1.0
```
1.2.0 doesn't exist at the moment.

If you're making an update for this package, only PackageVersion, URL, and Hash needs to be changed.